packages/meditrak-server/src/routes/editRecord.js

```diff
@@ -38,6 +38,7 @@ const EDITABLE_RECORD_TYPES = [
   TYPES.DASHBOARD_REPORT,
   TYPES.MAP_OVERLAY,
   TYPES.DASHBOARD_GROUP,
+  TYPES.PROJECT,
```

Comment:
@EMcQ-BES please add this to your list of conflicts (I guess it will need a whole card)